tests/test_crash_reports.py

```diff
@@ -161,29 +153,29 @@ def test_that_7_days_is_selected_default_for_nightlies(self, mozwebqa):
         top_crashers = csp.release_channels
         tc_page = top_crashers[1].click_top_crasher()
 
-        Assert.equal(tc_page.current_days_filter, '7')
+        assert 7 == tc_page.current_days_filter
 
     @pytest.mark.nondestructive
     def test_that_only_browser_reports_have_browser_icon(self, mozwebqa):
         csp = CrashStatsHomePage(mozwebqa)
         reports_page = csp.click_last_product_top_crashers_link()
         type, days, os = 'Browser', '7', 'Windows'
-        Assert.equal(reports_page.current_filter_type, type)
+        assert type == reports_page.current_filter_type
 
         reports_page.click_filter_days_by(days)
         reports_page.click_filter_os_by(os)
-        Assert.equal((type, days, os), (reports_page.current_filter_type,
-                                        reports_page.current_days_filter,
-                                        reports_page.current_os_filter))
+        assert (type, days, os) == (reports_page.current_filter_type,
+                                    reports_page.current_days_filter,
+                                    reports_page.current_os_filter)
```

Comment:
Could we split this into separate asserts? We're not saving anything by combining them.